phpBB/includes/functions_privmsgs.php

```diff
@@ -2109,6 +2109,7 @@ function message_history($msg_id, $user_id, $message_row, $folder, $in_post_mode
 			'MSG_ID'			=> $row['msg_id'],
 			'MESSAGE_TIME'		=> $row['message_time'],
 			'USER_ID'			=> $row['user_id'],
+			'USER_ID_NOT_SELF'	=> ($row['user_id'] != $user->data['user_id']) ? true : false,
```

Comment:
We consider true/false variables to be switches, and should start with an `S_`

This could be renamed more concisely to `S_NOT_SELF`

Reply:
@VSEphpbb According to guidelines, `S_` stands for `system`, not for `switch`.
[Coding Guidelines :: General Templating](https://area51.phpbb.com/docs/31x/coding-guidelines.html#templates)
> All template variables should be named appropriately (using underscores for spaces), language entries should be prefixed with L_, **system data with S_**...